src/styles/blocks/header.scss

```diff
@@ -38,5 +36,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $speed ease-out $speed;
+  }
+
+  &__link:hover {
+    transition: color $speed ease-out 0s;
+    color: #00acdc;
```

Comment:
It would be great to create variables for colors too, so you can change them in one place
![image](https://user-images.githubusercontent.com/96409563/229703368-b3d62885-d166-4d01-8452-6711bc15b70d.png)